backend/src/services/mentor/mentor.service.ts

```diff
@@ -44,6 +44,33 @@ class Mentor {
       userId,
     });
   }
+
+  public async checkIsMentor({
+    courseId,
+    userId,
+  }: CoursesToMentorsRequestDto): Promise<boolean> {
+    const isMentor = await this.#coursesToMentorsService.getByUserIdAndCourseId(
```

Comment:
- It's not boolean at this point of time
- We can improve performance of this query by not selecting anything, e.g `select 1 from table where condition`